rollastic/__main__.py

```diff
@@ -80,11 +84,19 @@ def upgrade(master_node, masters, datas, minimum_version):
         * Enable allocation
         * Wait until cluster is in green health
     '''
-    _LOG.info('Rolling upgrade with master_node=%s and minimum_version=%s', master_node, minimum_version)
+    # Assert that incompatible arguments are not specified, and determine hold policy
+    assert not (hold and unhold)
+    hold_package = None
+    if hold:
```

Comment:
There are 3 courses of action here. First, and the default, is to do nothing with the package mark. If neither --unhold or --hold are specified, we don't want to touch the package mark. However, if one is specified and not the other, we know that the user wants to override the package mark, and also either hold or unhold the package based on the parameter given. None is important because this value implies that no action should be taken on the package mark.